ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -155,6 +155,22 @@ tests:
     test:
     - ref: windows-e2e-operator-test-byoh
     workflow: windows-e2e-operator-test-platform-none-vsphere
+- as: vsphere-proxy-e2e-operator
+  optional: true
```

Comment:
`always_run` should be set to false as we expect these jobs to fail

Reply:
They should actually pass. Windows nodes function as normal, just ignoring the proxy
[successful job example](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/37634/rehearse-37634-pull-ci-openshift-windows-machine-config-operator-master-vsphere-proxy-e2e-operator/1638979745913769984)

Reply:
Why do we need to have this run on every PR until the feature is under development? We should be setting this `always_run` to `false`.

Reply:
Ah I misunderstood what `optional` did. I'll change this